extensions/trace-propagators/src/main/java/io/opentelemetry/extension/trace/propagation/B3Propagator.java

```diff
@@ -49,12 +50,16 @@
   static final String TRACE_ID_HEADER = "X-B3-TraceId";
   static final String SPAN_ID_HEADER = "X-B3-SpanId";
   static final String SAMPLED_HEADER = "X-B3-Sampled";
+  static final String DEBUG_HEADER = "X-B3-Flags";
   static final String COMBINED_HEADER = "b3";
   static final String COMBINED_HEADER_DELIMITER = "-";
+  static final String DEBUG_CONTEXT_KEY = "b3-debug";
+  static final ContextKey<String> DEBUG_CONTEXT = ContextKey.named(B3Propagator.DEBUG_CONTEXT_KEY);
```

Comment:
```suggestion
  static final ContextKey<String> DEBUG_CONTEXT_KEY = ContextKey.named("b3-debug);
```

The string form isn't really used, and it's the `ContextKey` that is actually the context key :)